README.md

```diff
@@ -77,4 +77,10 @@ Then you just need to start the harvester with:
     
 This will run through each account id in the API_KEYS list, and for each one retrieve the ISSNs from
 the DOAJ, then request the articles for each of those ISSNs from EPMC, and fire it over to the DOAJ
-(authenticating with that account's API key).
\ No newline at end of file
+(authenticating with that account's API key).
+
+## Rotating the logs
+
+The harvester creates a fair amount of log output, so it's best to rotate and archive old logs using ```logrotate```.
+Copy the config from ```deploy/logrotate/doaj-harvester``` into the logrotate directory at ```/etc/logorotate.d/``` and
```

Comment:
reloading configs not necessary, since it runs from cron.daily - it just picks up whatever's there next time it runs. I'll change copying to symlinking, though.